vms/pom/pages/volunteerProfilePage.py

```diff
@@ -23,9 +23,9 @@ def fill_values(self, new_details):
         self.fill_field(elements.PROFILE_LAST_NAME, new_details[1])
         self.fill_field(elements.PROFILE_EMAIL, new_details[2])
         self.fill_field(elements.PROFILE_ADDRESS, new_details[3])
-        self.fill_field(elements.PROFILE_CITY, new_details[4])
-        self.fill_field(elements.PROFILE_STATE, new_details[5])
-        self.fill_field(elements.PROFILE_COUNTRY, new_details[6])
+        self.send_value_to_xpath(elements.PROFILE_COUNTRY, new_details[6])
```

Comment:
Why change the order? Its anyhow going to be converted to dictionaries in my next week.

Reply:
Answered above